ingestion/src/test/java/feast/store/serving/redis/RedisCustomIOTest.java

```diff
@@ -202,9 +221,14 @@ public void shouldProduceFailedElementIfRetryExceeded() {
                         null))
             .collect(Collectors.toList());
 
+    StoreProto.Store store =
+        StoreProto.Store.newBuilder()
+            .setRedisConfig(redisConfig)
+            .setType(StoreProto.Store.StoreType.REDIS)
+            .build();
     PCollection<Long> failedElementCount =
         p.apply(Create.of(featureRowWrites))
-            .apply(RedisCustomIO.write(redisConfig))
+            .apply(RedisCustomIO.write(store))
```

Comment:
Is there a reason for this change? We will have a lot of duplicated code if each store has to figure out which config to use.

Reply:
I wanted to pass store into RedisCustomIO here so that I can check if the store is REDIS or REDIS_CLUSTER.

Reply:
```
     WriteDoFn(StoreProto.Store store) {
        if (store.getType() == StoreProto.Store.StoreType.REDIS)
          this.redisIngestionClient = new RedisStandaloneIngestionClient(store.getRedisConfig());
        else if (store.getType() == StoreProto.Store.StoreType.REDIS_CLUSTER)
          this.redisIngestionClient = new RedisClusterIngestionClient(store.getRedisClusterConfig());
      }
```

Reply:
Ok, makes sense.